flink-cdc-connect/flink-cdc-source-connectors/flink-connector-mysql-cdc/src/main/java/org/apache/flink/cdc/connectors/mysql/debezium/DebeziumUtils.java

```diff
@@ -240,12 +240,15 @@ private static Map<String, String> querySystemVariables(
         return variables;
     }
 
-    public static BinlogOffset findBinlogOffset(long targetMs, MySqlConnection connection) {
+    public static BinlogOffset findBinlogOffset(
+            long targetMs, MySqlConnection connection, MySqlSourceConfig mySqlSourceConfig) {
```

Comment:
How about passing the serverId directly? In this way the method signature is also clearer and easier to understand.

Reply:
At first, I did think so, but considering that this is a method inside the util attribute, there may be some changes later, and the attributes carried by the config class will create more in the future.